server-guide/jmap-server-guide.mdown

```diff
@@ -56,7 +56,7 @@ Data sync properties:
   (after stripping fwd/re etc.). This is used for checking whether we need to
   split out a new thread when a message arrives as the subject has changed (see data structure 7).
 - **messages**: `ThreadMessage[]`
-  The list of messages belonging to the thread, sorted with drafts immediately after the message they are in reply to, then by date order.
+  The list of messages belonging to the thread, sorted with drafts immediately after the message they are in reply to or then the message they are fowarding, then by date order.
```

Comment:
s/or then the/or the/
or better s/or then the message they are/or they are forwarding